modules/ui/src/main/scala/lucuma/ui/forms/Validate.scala

```diff
@@ -44,4 +45,21 @@ object InputValidate {
    */
   def fromIso[A](iso: Iso[String, A]): InputValidate[A] =
     fromFormat(Format.fromIso(iso))
+
+  /**
+   * Build optic from a Format but allow empty values to become `None
+   */
+  def fromFormatOptional[A](
```

Comment:
This is a last minute addition. It makes it possible for optional fields to be empty without triggering an error.